benefits/templates/admin/login.html

```diff
@@ -51,7 +51,7 @@
           <a href="{{ password_reset_url }}">Forgotten your login credentials?</a>
         </div>
       {% endif %}
-      <div class="submit-row">
```

Comment:
Changed this from `submit-row` to `login-submit-row` b/c Django has a class called `submit-row` already, and it adds unnecessary styles to this.